core/src/main/java/org/pentaho/platform/scheduler2/action/SchedulerOutputPathResolver.java

```diff
@@ -107,6 +133,16 @@ && isPermitted( outputFilePath ) ) {
     return runAsUser( callable );
   }
 
+  /**
+   * Combine <code>directory</code> and <code>filename</code>
+   * @param directory
+   * @param filename
+   * @return
+   */
+  public String concat( String directory, String filename ) {
```

Comment:
Could be private / protected.

Reply:
addressed via : https://github.com/pentaho/pentaho-scheduler-plugin/pull/60/commits/223802766d389eccdd3e76d6614bf03c1b625a3c